src/containers/LocationStorageContainer.tsx

```diff
@@ -108,6 +108,27 @@ export default class LocationStorageContainer extends Container<
     return this.saveLocations()
   }
 
+  deleteOrEditLocation = async (index: number, updatedLocation: object) => {
+    // get locations, remove the location by index
+    let locations = await SInfo.getItem(LOCATIONS_KEY, SINFO_OPTIONS)
```

Comment:
A couple of thoughts on this one:

- You shouldn't have to directly fetch and save the locations in this method. They should already be in `this.state.locations`, so you should be able to just call `await this.setState({locations: updatedLocations})` and then `return this.saveLocations()`.... like what `recordLocation` does.
- could we pass the timestamp instead of the index for deletion to avoid timing issues? (We could use a `Array.prototype.filter` to remove any entries matching the given timestamp, and it would avoid having to pass the index around in the LocationsScreen and LocationScreen.)
- Do we need to "update" locations? If so I'd rather have separate `deleteLocation` and `updateLocation` methods